system-test/common.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as uuid from 'uuid';
+import {Cluster} from '../src/cluster';
+import * as inst from '../src/instance';
+
+export const PREFIX = 'gcloud-tests-';
+
+export function generateId(resourceType: string) {
```

Comment:
same question here

Reply:
These variables were moved into a file called common.js because now two files use them. `app-profile.ts` is one of them and the other is `instance.ts`.